test/cookbooks/scponly-test/files/chroot.sh

```diff
@@ -0,0 +1,31 @@
+#! /bin/bash
+
+ALTROOT="$1"; shift
+BINARIES="$@"
+
+for b in $BINARIES
+do
+	d=$(dirname "${b}")
+	if [ ! -d "${ALTROOT}${d}" ]; then
+		/bin/mkdir -p "${ALTROOT}${d}"
+	fi
+	/bin/cp "${b}" "${ALTROOT}${d}"
+
+	LIBS=$(ldd "${b}" | awk '{ print $3 }' | egrep -v -v ^'\(')
+
+	for i in ${LIBS}; do
+		d=$(dirname "${i}")
+		if [ ! -d "${ALTROOT}${d}" ]; then
+			/bin/mkdir -p "${ALTROOT}${d}"
+		fi
+			/bin/cp "${i}" "${ALTROOT}${d}"
+	done
+
+	ld=$(ldd "${b}" | grep 'ld-linux' | awk '{ print $1 }')
+	d=$(dirname "${ld}")
+
+	if [ ! -d "${ALTROOT}${d}" ]; then
+		/bin/mkdir -p "${ALTROOT}${d}"
+	fi
+	/bin/cp "${ld}" "${ALTROOT}${d}"
+done
```

Comment:
Is this still needed?

Reply:
I believe so. ld-linux isn't listed in the same format as the other libraries so it isn't picked up by the other loop above and it has to be copied over this way.

Reply:
I'm just confused why this is in the test cookbook when we already have something like it in the actual cookbook.

Reply:
It's just a link. For some reason when I tried to source the script in the resource it looked in the the test cookbook files directory not the base cookbook. Without the link it was giving me an error saying the file couldn't be found